### ISSUE TITLE
Add additional audio effects

### ISSUE THREAD

**Issue by LordRaptor** [alfraprog/davincibros#16]
- [ ] Pistol fire
- [ ] Pistol hit
- [ ] Arrow launch
- [ ] Arrow hit
- [ ] Grenade launch
- [ ] Screw flying (should support continous sound)
- [ ] Spring flying
- [ ] Rocket flying (should support continous sound)
- [ ] Tank landing on the ground

**Comment by alfraprog** [comment]
I can do that! i’ll see what is the easiest for fly/impact logic in fmod to make it flexible for future cases

**Comment by alfraprog** [comment]
tank landing on ground probably needs to vary based on tires

**Comment by LordRaptor** [comment]
You are right, so the tires can actually handle spawning the sound. Once you've created the sounds, I'll create an issue to integrate them
